dashboard/app/models/game.rb

```diff
@@ -51,6 +51,10 @@ def self.custom_maze
   LEVEL_GROUP = 'level_group'.freeze
   PUBLIC_KEY_CRYPTOGRAPHY = 'public_key_cryptography'.freeze
 
+  def self.flappy
+    @@game_flappy ||= find_by_name("Flappy")
```

Comment:
Ah, sorry!  It means that any Flappy level will always have `flappy_intro` as an autoplay video.  Via [LevelsHelper#select_and_track_autoplay_video](https://github.com/code-dot-org/code-dot-org/blob/07fdda15d346db7a4b091b8f3ec00531dac39406/dashboard/app/helpers/levels_helper.rb#L84) -> [Level#related_videos](https://github.com/code-dot-org/code-dot-org/blob/1fd7c7208d78c037860d3ea3b92b810a7d4546ea/dashboard/app/models/levels/level.rb#L90).

```ruby
def related_videos
  ([game.intro_video, specified_autoplay_video] + concepts.map(&:video)).compact.uniq
end

autoplay_video = @level.related_videos.find {|video| !client_state.video_seen?(video.key)}
```